Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -876,15 +864,32 @@ internal void AfterUpdateScene()
             MakeSureAssetDirectoryExists();
             Texture2D texture = null;
 #if UNITY_EDITOR
-            Action<string> doImport = (path) =>
-            {
+            Action<string> doImport = (path) => {
+                bool assetExisted = AssetDatabase.LoadAssetAtPath<Texture2D>(path) != null;
+
                 AssetDatabase.ImportAsset(path);
                 texture = AssetDatabase.LoadAssetAtPath<Texture2D>(path);
                 if (texture != null) {
                     TextureImporter importer = (TextureImporter)AssetImporter.GetAtPath(path);
+
+                    // Make sure the full texture is used unless the user changed this setting before:
+                    assetExisted = true;
+                    if (!assetExisted) {
```

Comment:
This seems like a new change.
`assetExisted` is always true here, isn't it ?
Can we remove the branching ?

Reply:
`assetExisted` would be false if the file was just saved and `AssetDatabase.ImportAsset` has not been called yet.

Reply:
Hmm ? `assetExisted` is set to true just before the branching, which means that the branching isn't really being used, isn't it ?

Reply:
I've removed that now.